src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -54,6 +54,47 @@ public OCFile createFromParcel(Parcel source) {
     public static final String PATH_SEPARATOR = "/";
     public static final String ROOT_PATH = PATH_SEPARATOR;
 
+    public enum FavoriteStatus {
```

Comment:
This is a great opportunity to stop calling it "Favorite" also internally, and start calling it "AvailableOffline".